oshi-core/src/main/java/oshi/software/os/OperatingSystem.java

```diff
@@ -213,7 +213,7 @@ default List<OSProcess> getProcesses() {
      *         specified process ids if it is running
      */
     default List<OSProcess> getProcesses(Collection<Integer> pids) {
-        return pids.stream().map(this::getProcess).filter(Objects::nonNull).filter(ProcessFiltering.VALID_PROCESS)
+        return pids.stream().parallel().map(this::getProcess).filter(Objects::nonNull).filter(ProcessFiltering.VALID_PROCESS)
```

Comment:
Probably doesn't hurt, but also would give nondeterministic ordering of results when a user would expect the results in the same order of a list (if a list was the input).  So no.